server/eclipse-project/src/main/java/us/freeandfair/corla/asm/AsmEvent.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Free & Fair Colorado RLA System
+ * 
+ * @title ColoradoRLA
+ * @created Aug 8, 2017
+ * @copyright 2017 Free & Fair
+ * @license GNU General Public License 3.0
+ * @author Joe Kiniry <kiniry@freeandfair.us>
+ * @description A system to assist in conducting statewide risk-limiting audits.
+ */
+
+package us.freeandfair.corla.asm;
+
+/**
+ * The events of the Abstract State Machine (ASM) of the Colorado RLA Tool.
+ * @trace asm.asm_event
+ * @author Joseph R. Kiniry <kiniry@freeandfair.us>
+ * @version 0.0.1
+ */
+public interface AsmEvent {
+  /**
+   * The Department of State Dashboard's events.
+   * @trace asm.department_of_state_dashboard_event
+   */
+  enum DosDashboardEvent implements AsmEvent {
+    AUTHENTICATE_STATE_ADMINISTRATOR_EVENT,
+    ESTABLISH_RISK_LIMIT_FOR_COMPARISON_AUDITS_EVENT,
+    SELECT_CONTESTS_FOR_COMPARISON_AUDIT_EVENT,
+    PUBLIC_SEED_EVENT,
+    PUBLISH_BALLOTS_TO_AUDIT_EVENT,
```

Comment:
These are out of order. It is imperative that  PUBLISH_BALLOTS_TO_AUDIT_EVENT comes before PUBLIC_SEED_EVENT